src/main/java/com/faforever/client/util/DateFormatterUtil.java

```diff
@@ -0,0 +1,56 @@
+package com.faforever.client.util;
+
+import lombok.NoArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.time.format.DateTimeFormatter;
+import java.time.format.FormatStyle;
+
+@Service
+@NoArgsConstructor
+public class DateFormatterUtil {
+
+  public static final DateTimeFormatter FORMATTER_AUTO_SHORT = DateTimeFormatter.ofLocalizedDate(FormatStyle.SHORT);
+  public static final DateTimeFormatter FORMATTER_AUTO_MEDIUM = DateTimeFormatter.ofLocalizedDate(FormatStyle.MEDIUM);
+  public static final DateTimeFormatter FORMATTER_AUTO_LONG = DateTimeFormatter.ofLocalizedDate(FormatStyle.LONG);
+  public static final DateTimeFormatter FORMATTER_AUTO_FULL = DateTimeFormatter.ofLocalizedDate(FormatStyle.FULL);
+
+  public static final DateTimeFormatter FORMATTER_DMY_SHORT = DateTimeFormatter.ofPattern("d/M/yy");
+  public static final DateTimeFormatter FORMATTER_DMY_MEDIUM = DateTimeFormatter.ofPattern("d MMM, yyyy");
+  public static final DateTimeFormatter FORMATTER_DMY_LONG = DateTimeFormatter.ofPattern("d MMMM, yyyy");
+  public static final DateTimeFormatter FORMATTER_DMY_FULL = DateTimeFormatter.ofPattern("EEEE, d MMMM, yyyy");
+
+  public static final DateTimeFormatter FORMATTER_MDY_SHORT = DateTimeFormatter.ofPattern("M/d/yy");
+  public static final DateTimeFormatter FORMATTER_MDY_MEDIUM = DateTimeFormatter.ofPattern("MMM d, yyyy");
+  public static final DateTimeFormatter FORMATTER_MDY_LONG = DateTimeFormatter.ofPattern("MMMM d, yyyy");
+  public static final DateTimeFormatter FORMATTER_MDY_FULL = DateTimeFormatter.ofPattern("EEEE, MMMM d, yyyy");
```

Comment:
These should be private and can just be members of TimeSerivce rather than being in a separate class

Reply:
I don't see advantage to use them as members. From memory footprint, it will still be created anew( afterwards dateformatter is called by other methods that create its copy) and then we use new one all over again for AUTO?

Though i fixed as you asked.

Reply:
It also just helps communicate the intent of these as constants